src/core/providers/ogr/qgsogrproviderutils.cpp

```diff
@@ -1314,6 +1314,11 @@ QByteArray QgsOgrProviderUtils::quotedIdentifier( QByteArray field, const QStrin
     field.replace( '`', "``" );
     return field.prepend( '`' ).append( '`' );
   }
+  else if ( driverName == QLatin1String( "GPKG" ) || driverName == QLatin1String( "SQLite" ) || driverName == QLatin1String( "SpatiaLite" ) )
```

Comment:
there's no spatialite driver. It is a sub-mode of the SQLite one:
```suggestion
  else if ( driverName == QLatin1String( "GPKG" ) || driverName == QLatin1String( "SQLite" ) )
```

Reply:
@rouault , thanks.